plugin.yml

```diff
@@ -1,8 +1,8 @@
 ---
 name: ProfanityFilter
-version: 0.0.8
+version: 0.0.9-BETA
 main: ReinfyTeam\ProfanityFilter\Loader
-api: 4.0.0
+api: [4.0.0,5.0.0]
```

Comment:
Does Poggit allows this?
4.0.0 and 5.0.0 at the same time

Reply:
I'll ask later...